pkg/network/network.go

```diff
@@ -23,6 +23,7 @@ func Validate(conf *opv1alpha1.NetworkAddonsConfigSpec, openshiftNetworkConfig *
 
 	errs = append(errs, validateMultus(conf, openshiftNetworkConfig)...)
 	errs = append(errs, validateImagePullPolicy(conf)...)
+	errs = append(errs, validateKubeMacPool(conf)...)
```

Comment:
please move above image pull policy.

i'd like to keep image pull policy on bottom everywhere since it is not a component like all the others, it's just additional lifecycle parameter